deployments/container/reconfigure-mig.sh

```diff
@@ -525,28 +541,31 @@ if [ "${?}" != "0" ]; then
 fi
 
 if [ "${CDI_ENABLED}" = "true" ]; then
-	echo "Running nvidia-smi"
-	chroot ${DRIVER_ROOT_CTR_PATH} nvidia-smi >/dev/null
-	if [ "${?}" != "0" ]; then
-		exit_failed
-	fi
-
 	echo "Creating NVIDIA control device nodes"
-	nvidia-ctk system create-device-nodes --control-devices --driver-root=${DRIVER_ROOT_CTR_PATH}
+	nvidia-ctk system create-device-nodes \
+	  --load-kernel-modules \
```

Comment:
This won't work currently since the toolkit code to load kernel modules relies on chroot'ng into the driverRoot.

General question -- does anyone remember why it was required to run nvidia-smi first before creating the device nodes and generating the management CDI spec?

Reply:
I have removed `--load-kernel-modules` for the time being since this currently does not work when driverRoot is not chroot'ble.

Reply:
I think we use `nvidia-smi` to create the NON control device nodes. I don't believe it's required to run `nvidia-smi` BEFORE we run this command, but we do need to run it in addition to running this if we wanted the device nodes to be created.

The issue with `nvidia-smi` is that it doesn't create the `/dev/nvidia-uvm*` devices nodes, which is why we create them here.

Reply:
The non-control devices should typically exist at this point, except in the scenario where a reboot is required to toggle the MIG mode and the driver is preinstalled. Even though the driver and toolkit validations run before mig-manager starts, the invocation of nvidia-smi will create the device nodes in the container's dev and not the host's.

Any recommendations for how we should handle this case?

Reply:
I have updated the script to run `nvidia-smi` first if the NVIDIA driver is installed on the host (e.g. at `/`). This should address the scenario I described in https://github.com/NVIDIA/mig-parted/pull/69#discussion_r1625372657